Issue Number 9.R

```diff
@@ -0,0 +1,22 @@
+
+survey_results_public <- read.csv("C:/Users/Vinesh/Dropbox/DIT/Higher Diploma/Year 2/Group Project/survey_results_public.csv")
+

```

Comment:
can also use read_csv in the ready library

Reply:
:+1: